### PR TITLE
V2.2.27.27

### LINKED ISSUE #35: Maintainer changed account
I think the maintainer Peter Killick changed their account, which irritates the bots.

@DPeterK, can you confirm that you used to be @dkillick and change the information in the recipe accordingly?

### LINKED ISSUE #36: Version update
Issue: Some new releases of udunits2 are available. Specifically, I need to be able to format timeunits including "since 2001-01-01 00:00:00", something that was added in Unidata/UDUNITS-2@3e607e13e8964e21e358690fa87b5e8a03b140e1 and released in version 2.2.27.15 (Unidata/UDUNITS-2@a710228a8a02d46e8058507487b33299cc1de7cf). Of course, an update to the most recent version seems sensible.

I am happy to work on this; since there is already a corresponding bot PR, it would probably make sense to join the team before starting on it. Or do you prefer a PR against the open PR?

ping @conda-forge/udunits2

### PR DESCRIPTION
**Pull request by zklaus**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Resolves #35, resolves #36 
The following PRs are made obsolete by this one:
Closes #33, closes #34 

This PR removes the windows build to make it work as-is. This is not ideal, but note that the windows support has been absent for all 2.2.27.x builds. In light of that, I think it's better to update the working builds and tackle windows support in a separate PR.

### CODE REVIEW

**Comment by zklaus** [comment]
@conda-forge-admin, please rerender

**Comment by zklaus** [comment]
This PR removes Windows support, which is not ideal. However, note that Windows support has been absent for all of the 2.2.27 builds. In light of that, I think it's better to update the available version where possible and then tackle the adding of windows support in another PR.

**Comment by zklaus** [comment]
@conda-forge/udunits2, I think this is ready for review.

**Comment by zklaus** [comment]
I sought clarification of the license status in Unidata/UDUNITS-2#102 and it seems the license is not OSI approved after all. Hence, I removed the corresponding note.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
   url: https://github.com/Unidata/UDUNITS-2/archive/v{{ version }}.tar.gz
-  sha256: 74fd7fb3764ce2821870fa93e66671b7069a0c971513bf1904c6b053a4a55ed1
+  sha256: 6c65b3097ef38b94c01dfede8904346c1439de9704b9bae7deea72bd5143a7bb
 
 build:
-  number: 1001
+  number: 0
   detect_binary_files_with_prefix: true
-  skip: True  # [win and vc<14]
+  skip: True  # [win]
```
Comment (ocefpaf) [inline recipe/meta.yaml:14]: Let's not skip Windows all together. I'd rather merge a PR with a failing Windows CI and open an issue to follow up than just remove the problem from sight.

Comment (zklaus) [reply recipe/meta.yaml:14]: Ok, I will reenable the windows build. I thought a failing CI would be a hard blocker, but I agree with this approach.
Note, however, that upstream seems unable to build on Windows, see Unidata/UDUNITS-2#96, Unidata/UDUNITS-2#87.

**Comment by zklaus** [comment]
@conda-forge-admin, please rerender

**Comment by zklaus** [comment]
Seems the only thing missing to make the Windows build work was to create the `%SCRIPTS%` directory.

**Comment by ocefpaf** [comment]
Awesome! Thanks @zklaus!!

**Comment by zklaus** [comment]
Cheers :smile:

**Pull request merged**